commands/mr/update/mr_update.go

```diff
@@ -189,6 +217,7 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 	mrUpdateCmd.Flags().StringSliceP("label", "l", []string{}, "add labels")
 	mrUpdateCmd.Flags().StringSliceP("unlabel", "u", []string{}, "remove labels")
 	mrUpdateCmd.Flags().StringSliceP("assignee", "a", []string{}, "assign users via username, prefix with '!' or '-' to remove from existing assignees, '+' to add, otherwise replace existing assignees with given users")
+	mrUpdateCmd.Flags().StringSliceP("reviewer", "", []string{}, "assign users via username, prefix with '!' or '-' to remove from existing reviewers, '+' to add, otherwise replace existing reviewers with given users")
```

Comment:
The flag description for `--reviewer` could be better as `add reviewers ...` instead of `assign users ...`
```suggestion
	mrUpdateCmd.Flags().StringSliceP("reviewer", "", []string{}, "add reviewers via username, prefix with '!' or '-' to remove from existing reviewers, '+' to add, otherwise replace existing reviewers with given users")
```